prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1428,7 +1428,7 @@ int rxr_ep_init(struct rxr_ep *ep)
 
 	if (rxr_env.rx_copy_unexp) {
 		ret = ofi_bufpool_create(&ep->rx_unexp_pkt_pool, entry_sz,
-					 RXR_BUF_POOL_ALIGNMENT, 0,
+					 ofi_get_page_size(), 0,
```

Comment:
Only tx and rx packet pool's memory are registered. so the other does not need change.